resources/teamcity/developer/publish-developer-to-downloads-keyman-com.ps1

```diff
@@ -54,25 +50,14 @@ mkdir ${upload_path}
 #
 
 if((Test-Path ..\release\${kmcomp_zip}) -ne 0) {
+  # REVIEW: I guess this is the path for Keyman 18+. In which case we
+  # can remove the else block
   copy ..\release\${kmcomp_zip} ..\${upload_path}\${kmcomp_zip}
```

Comment:
I don't think this is the path for Keyman 18+. I think this is old and should be removed; it looks to me like kmcomp.zip is only built in the else block below.